spec.html

```diff
@@ -78,8 +78,10 @@ <h2>Shape Tree Structure <a class="cross" href="primer#structure"></a></h2>
   (rdfs:label xsd:string | tree:uriTemplate xsd:string) ;
   tree:uses IRI ? ;
   (tree:shape IRI | tree:mediaType xsd:string )? ;
-  tree:contents @&lt;#Step&gt;*
+  tree:contents @&lt;#Step&gt;* 
 }
+
+<> skosIndex <./MyKnownInterfaceHierarchies> .
```

Comment:
I'm not sure that we need to include the skosIndex statement in this specific example? Also ./MyKnownInterfaceHierarchies is a bit of a mouthful but at the same time, isn't very intuitive to the layman.

Reply:
Agreed on both points.  Removed.